### ISSUE TITLE
Jade is required for e2e tests

### ISSUE THREAD

**Issue by CAYdenberg** [keystonejs/keystone#4340]
### Expected behavior

After `npm run test-all` on the master branch with no changes, the tests should all pass

### Actual/Current behavior

```sh
Running:  AdminUI app should allow navigating to the front page by clicking the Front Page Icon
Error thrown for request: /
Error: Cannot find module 'jade'
```

If I install jade the tests all pass. Perhaps it should be in devDependencies?

### Steps to reproduce the actual/current behavior

Detailed above

### Environment

| Software         | Version
| ---------------- | -------
| Keystone         |  Current master branch (fe4be86)
| Node             |  v6.6.0

**Comment by jstockwin** [comment]
Hey, yeah I noticed this too, but there are also other issues with the testing and I was (and still am) too tied up in exams to check them. 

To me, it seems silly to include jade in the dependencies just for the sake of running the tests. The only reason it's there is to check the `back to main site` link works, and so there is an index page of the main site that uses jade. 

I think the proper fix for this, rather than installing jade, would be to re-write this page as a static page so as not to use jade. To be honest, it would probably be good enough to simply copy and paste the page source. There is no dynamic content on the page, so it might as well be static. 

I hope that makes sense.

**Comment by jstockwin** [comment]
@CAYdenberg I've self assigned this so I don't forget about it, but if you want to implement a fix then please go ahead. If you do, I'll review your PR next week (after exams). If not, I'll get on it myself. Thanks!

**Comment by CAYdenberg** [comment]
I'm not too familiar with this type of testing but I'll take a look.

**Comment by jstockwin** [comment]
It's not really anything to do with the testing itself. Jade is only used in `test/e2e/frontend/index.jade`, so if we can get that changed to `index.html` or something which doesn't use jade then we should be done. 

The `test/e2e` folder is essentially its own keystone instance, with `server.js` instead of the usual `index.js`, and it has all the usual `routes` folders etc. 

Feel free to have a look, but no problem if not as I can do it next week.

**Comment by CAYdenberg** [comment]
The tests pass on my machine but Travis says "deploy preview failed". Maybe you can help when you're time frees up?

**Comment by jstockwin** [comment]
Yeah, so the issue is that the location field is failing on travis but not locally. I tried to have a look in to this a few weeks ago, but didn't get anywhere. For some reason clicking the "show more" button on field B isn't working in the travis/saucelabs environment - it's one of those problems that is hard to debug.

Why don't we just disable the test altogether, for now, as it appears to be non-trivial to fix. 

You can disable the test by doing the equivalent of [this line](https://github.com/keystonejs/keystone/blob/45b157a8e6e776a967ba3beccbd103d0cef22eee/test/e2e/adminUI/tests/group006Fields/testSelectField.js#L5) but for `testLocationField.js`. 

After that tests should be passing and I'll merge, then fix the location tests at a later date.

Note to self: Open an issue about location test being disabled when PR gets merged.

**Comment by jstockwin** [comment]
@CAYdenberg Will you be able to disable that test? It's in your branch so I can't do it on top of your work with removing Jade, and it makes sense for it all to be merged together.

**Comment by CAYdenberg** [comment]
Done - hopefully that works

**Comment by jstockwin** [comment]
Perfect, thanks so much! Fingers crossed.

**Comment by jstockwin** [comment]
This was fixed in #4346, thanks @CAYdenberg
